### PR TITLE
add amset, boltztrap2, orthopy, ndim and quadpy  

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
**Two remarks:**

* `amset` requires `boltztrap2` but `boltztrap2` is not available for windows, so the windows build fails. 
* The latest version of `amset` require `scipy` 1.5.4 so I submit amset 0.3.0 here and plan to update it as soon as `scipy` 1.5.4 is available.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/boltztrap2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+test:
+  imports:
+    - BoltzTraP2
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.imc.tuwien.ac.at//forschungsbereich_theoretische_chemie/forschungsgruppen/prof_dr_gkh_madsen_theoretical_materials_chemistry/boltztrap/
```
Comment (ocefpaf) [inline recipes/boltztrap2/meta.yaml:49]: There are two slashes in there.

**Review by ocefpaf — COMMENTED** [review]
One minor fix and we can merge this.

File: recipes/boltztrap2/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/boltztrap2/meta.yaml]: Done

**Pull request merged**